client/src/store/story.ts

```diff
@@ -11,3 +13,66 @@ export const categoryAtom = atom<string>({
     }),
   ],
 });
+
+export const filtersOpenAtom = atom<boolean>({
+  key: 'filtersOpen',
+  default: false,
+});
+
+export const tagsAtom = atom({
+  key: 'tags',
+  default: [],
+  effects: [
+    urlSyncEffect({
+      refine: array(string()),
+    }),
+  ],
+});
+
+export const ifiAtom = atom({
+  key: 'ifi',
+  default: [],
+  effects: [
+    urlSyncEffect({
+      refine: array(string()),
+    }),
+  ],
+});
+
+export const statusAtom = atom({
+  key: 'status',
+  default: [],
+  effects: [
+    urlSyncEffect({
+      refine: array(string()),
+    }),
+  ],
+});
+
+export const useFilters = () => {
```

Comment:
It would be better id you use a selectorFamiliy instead of a custom way of getting/setting filters. Here you have an example =>
- https://github.com/Vizzuality/foodscapes/blob/develop/client/src/store/explore-map/index.ts#L239
- https://recoiljs.org/docs/api-reference/utils/selectorFamily/